# Message describing the command joint angles
Header header
float32[] angles